cves/kernel/CVE-2019-13648.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: | 
+  On PowerPC Machines, local users can trick the system into attempting to use 
+  an unsupported/disabled feature which causes a system crash. The user can set 
+  bits of the MSR(Machine-State Register) to allow HTM(Hardware Transactional Memory), even 
```

Comment:
Looks better if you have spaces before the parenthesis. Ex. "MSR (Machine-State Register)" instead of "MSR(Machine-State Register)"

Reply:
Fixed